pkg/cmd/pipeline/create.go

```diff
@@ -0,0 +1,127 @@
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/buildkite/cli/v3/internal/io"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/buildkite/go-buildkite/v3/buildkite"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdPipelineCreate(f *factory.Factory) *cobra.Command {
+
+	cmd := cobra.Command{
+		DisableFlagsInUseLine: true,
+		Use:                   "create",
+		Short:                 "Creates a new pipeline",
+		Args:                  cobra.MaximumNArgs(1),
+		Long: heredoc.Doc(`
+			Creates a new pipeline in the current org and outputs the URL to the pipeline.
+
+			The pipeline can be a {pipeline_slug} or in the format {org_slug}/{pipeline_slug}.
+			If the pipeline argument is omitted, the user will be prompted for a name and description.
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var repoURL string
+
+			qs := []*survey.Question{
```

Comment:
The survey package we are using was deprecated in April!! 😢 

We can keep using it for now and follow up with replacing it. On the readme it says to use bubbletea 😆 

One cool thing we'll be able to do with that is use the [huh package hide feature](https://github.com/charmbracelet/huh/blob/5b41f0b45136a7a93eaaf25894dd8bd0f44e510c/group.go#L138). That will gracefully allow us to accept flags or arguments to this command and build up a form to get the other fields from the user.